lib/benchmarker.rb

```diff
@@ -0,0 +1,218 @@
+$LOAD_PATH.unshift File.expand_path('../', __FILE__)
```

Comment:
This testing is more elaborate than I anticipated. Why are you going through the HTTP stack here with Toxiproxy? You're using a bunch of test helpers, if you want to use them, instead of duplicating the code can you create helpers that you can include here if you want to continue to use the `Net::HTTP` stack?

Also, we should move this to `/test/benchmark` directory instead—`lib/` is going to be included with the gem, which we definitely don't want :)

Reply:
I mainly wanted to replicate a real-life environment but I might have gone too far and that's most likely not needed. I'll move everything into a helper! 👍